tests/componenttests/client/CMakeLists.txt

```diff
@@ -102,6 +103,7 @@ target_include_directories(
         $<TARGET_PROPERTY:RialtoPlayerPublicClientMocks,INTERFACE_INCLUDE_DIRECTORIES>
         $<TARGET_PROPERTY:RialtoTestCommonMatchers,INTERFACE_INCLUDE_DIRECTORIES>
         $<TARGET_PROPERTY:RialtoTestCommonProtoUtils,INTERFACE_INCLUDE_DIRECTORIES>
+        $<TARGET_PROPERTY:RialtoServerMain,INCLUDE_DIRECTORIES>
```

Comment:
Move the keysystems to playercommon so we dont have to include server side main code.